website/content/en/docs/best-practices/common-recommendation.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Common recommendations and suggestions"
+linkTitle: "Common suggestions"
+weight: 2
+description: Common recommendations and suggestions to built solutions with Operator SDK
+---
+
+## Overview
+
+Any recommendations or good practices suggested by the Kubernetes community, such as to develop [Operator pattern][operator-pattern] solutions or to use [controller-runtime][controller-runtime], also are good recommendations for those who are looking to build the operator projects with SDK. Also, see [Operator Best Practices][operator-best-practices].
+
+## Common Recommendations
+
+### Develop idempotent reconciliation solutions
+
+See that following the operator's pattern. You will create controllers, from [controller-runtime][controller-runtime], with a reconcile function responsible for synchronizing the resources until there's desired state in the cluster. So, it is recommended to develop idempotent solutions for your reconciliations.
```

Comment:
You don't mention idempotency until the end, i would lead with it. something like:

"When developing operators, it is important for the controllers reconciliation loop to be idempotent. [Controller-runtime's][controller-runtime] Reconcile function is responsible for synchronizing resources until the desired state has been reached, and can be invoked with the object in any possible actual and desired state. If your reconciliation loop is not idempotent, it may be possible for your controller or a watched resource to become "stuck" and require manual intervention.